src/main/java/com/finance/app/infrastructute/exception/ContentNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package com.finance.app.infrastructute.exception;
+
+public class ContentNotFoundException extends RuntimeException {
+
+    private String value;
```

Comment:
why do you create such exception ? Hope you know that it will be deserialized wrongly .